cmd/tracee-ebpf/flags/capabilities_test.go

```diff
@@ -1,88 +0,0 @@
-package flags_test
-
-import (
-	"testing"
-
-	"github.com/aquasecurity/tracee/cmd/tracee-ebpf/flags"
-	"github.com/stretchr/testify/assert"
-	"kernel.org/pub/linux/libs/security/libcap/cap"
-)
-
-func TestPrepareCapsConfig(t *testing.T) {
```

Comment:
I think this test was pretty easy to adjust to the new logic which is even simpler. 
At the very least let's open an issue to add it back, can be marked as a "good first issue" too.

Reply:
https://github.com/aquasecurity/tracee/issues/2258